### ISSUE TITLE
Accessibility Challenge

### ISSUE THREAD

**Issue by HStewart23** [fac-15/team-1-portfolio#29]
- Please explain ICONS & use of i tag (do you need alt text for icons?) 
- use of <br> is not good semantic design. Use CSS for breaks and spacing
- Footer text should be within another tag, not just within footer tag

##Easy to Read

- code :
     html & css: really nicely organised, easy to read and concise 
- CSS should use more comments to show what code belongs where (examples: nav style, description style etc.)
- some CSS rules are overwritten (use console in chrome to see)

## Navigation with Screen Reader
- Good job on screen reader! We only found a few things ...
- home button does not have id that identifies it as also a home button. On a screen reader you cannot tell what it does.
- Screen reader reads input box names twice if you click on the form as a whole possibly because they have both a 'for'  attribute as well as an id. The individual boxes work really nicely. 
- The screen reader does not read anything out when you click the submit button. Possibly because it does not have a label. 

## Simelar experience with/without javascript.
- no javascript so experience is identical.

**Comment by matthall8** [comment]
🎨 🎨 Top GitHub tip: Use back tics (`) when inputting code onto GitHUB e.g. `<br />` to ensure it can still be seen once posted 🎨 🎨

See https://help.github.com/articles/creating-and-highlighting-code-blocks/ for more details.

**Comment by Oliversw** [comment]
1. Icons are part of [Font-Awesome](https://fontawesome.com) and the ```<i>``` tags are how you the website suggests that you refer to them. Matt suggested somewhere replacing them with spans, so we need to look into that!
We had a look online but couldn't find anything about including alt tags on icons, and have opted to leave them out because they are more decorative than descriptive. All of the information about what they refer to should (ideally!) be in the text. Hope that's clear!
2. Good point on the ```<br>``` tags, that's something that I didn't know about, thanks! We removed them and fixed the contact form with CSS by using flexbox.

**Comment by jcharnley** [comment]
added p tag to the footer
CSS comments now included
home button now identified and moves to the home area with clicked
added named to submit input tag so it reads out when selected
